packages/skeleton-common/src/classes/floating-panel.ts

```diff
@@ -0,0 +1,15 @@
+import { defineSkeletonClasses } from '../internal/define-skeleton-classes.js' with { type: 'macro' };
+
+export const classesFloatingPanel = defineSkeletonClasses({
+	trigger: '',
+	positioner: '',
+	content: '',
```

Comment:
Zag has the right idea with their examples to allow scrolling when you hit minimal breakpoints. But they didn't account for the header bar - so icons actually break out of their container.

<img width="373" height="129" alt="Screenshot 2025-11-18 at 10 12 38 AM" src="https://github.com/user-attachments/assets/81ce2163-34b0-481e-ba66-329dba02ad22" />

<img width="369" height="412" alt="Screenshot 2025-11-18 at 10 19 55 AM" src="https://github.com/user-attachments/assets/b6105ed9-cba4-4805-8e1a-342d12a17fce" />

---

This is untested, but I'd expect something like this to work:

```
header: w-full overflow-x-hidden
```
```
content: overflow-x-auto overflow-y-auto
```